pom.xml

```diff
@@ -84,6 +84,7 @@
         <java.xml.ws.version>2.3.1</java.xml.ws.version>
         <jcodings.version>1.0.55</jcodings.version><!-- used by hbase -->
         <joni.version>2.1.31</joni.version><!-- used by json-validator -->
+        <jsoup.version>1.12.1</jsoup.version><!-- used by oaipmh -->
```

Comment:
camel-oaipmh brought 1.13.1 in compile scope while quarkus-junit5 brought 1.12.1 in test scope. So I aligned on quarkus version with compile scope.